app/controllers/admin/sitemaps_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Admin::SitemapsController < Admin::AdminController
+  active_scaffold :sitemap do |config|
+    config.label = "Sitemaps"
+    config.actions = [:create, :delete, :list]
+    config.columns = [:url, :created_at, :updated_at]
+    config.create.columns = [:url]
+    config.create.link.label = "Create Sitemap"
```

Comment:
You can skip the custom label here to be consistent with the implementation in other pages/sections in super admin, which just display the default 'Create New' label.